.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,27 @@
+# .github/workflows/chromatic.yml
+
+name: 'Chromatic'
+
+on:
+  pull_request_target:
+    types: [opened, reopened, ready_for_review]
+
+jobs:
+  chromatic:
+    name: Run Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install dependencies
+        run: npm i
+      - name: Run Chromatic
+        uses: chromaui/action@latest
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

Comment:
So, we need to create a project token from AsyncAPI account, right @devilkiller-ag ?

cc: @derberg

Reply:
We need to create an account on Chromatic and then add the website repository as a new project on it. Then we need to grab the CHROMATIC_PROJECT_TOKEN from the project settings. We need to set the `CHROMATIC_PROJECT_TOKEN` environment variable in the website's repository gitHub actions.

Reply:
`CHROMATIC_PROJECT_TOKEN` created. For now we are on free plan, but I also applied for open source plan

Reply:
we won't get open source plan at the moment, but free tier should be fine

lemme know if you need something more